src/components/ReportForm.js

```diff
@@ -136,8 +213,9 @@ class ReportForm extends React.Component {
    * State of form components.
    */
   state = {
-    date: formatDate(new Date()),
-    time: '00:00',
+    month: '01',
+    year: getYear(new Date()),
+    time: 'morning',
```

Comment:
Do you think the default time should be 'unknown', I feel like I remember him mentioning that
```suggestion
    time: 'unknown',
```

Reply:
You're probably right. I'll fix that.